src/Fast.xs

```diff
@@ -233,6 +233,17 @@ void Redis__Cluster__Fast_run_cmd(pTHX_ Redis__Cluster__Fast self, int argc, con
             return;
         }
         redisClusterAsyncDisconnect(self->acc);
+
+        if (event_base_dispatch(self->cluster_event_base) == -1) {
+            reply_t->error = newSVpvf("%s", "event_base_dispatch failed after forking");
+            return;
+        }
+
+        if (redisClusterConnect2(self->acc->cc) != REDIS_OK) {
```

Comment:
[MEMO]

I implemented (1.) as it allows for a more simple solution.

> Issue event_reinit and redisClusterAsyncDisconnect, then use redisClusterConnect2 to clear the shutdown flag and reconnect.

> Since there’s no significant difference, either method should work.